### PR TITLE
updates theme.json to v2 for compatibility with filtering

### PR DESCRIPTION
**Pull request by jacobdubail**
### Description of the Change

This updates the theme.json version to v2 for compatibility with the new theme.json filters. It also updates keys to match v2 schema.

### How to test the Change

Confirm custom styles are applying to the theme.

### Checklist:
- [ ] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: themes/10up-theme/theme.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-  "version": 1,
+  "$schema": "https://schemas.wp.org/wp/6.0/theme.json",
```
Comment (fabiankaegy) [inline themes/10up-theme/theme.json]: We should use 6.1 at this point :)

**Review by fabiankaegy — APPROVED** [review]
Thanks for the update! Added one comment that I think should be changed before merging but otherwise good to go 👍

File: themes/10up-theme/theme.json (hunk 1 above)
Comment (jacobdubailcadmv) [reply themes/10up-theme/theme.json]: Thanks for catching this! I pushed a new commit that updates the schema to 6.1

**Review by fabiankaegy — APPROVED** [review]

**Comment by fabiankaegy** [comment]
@jacobdubail If you want to you can also change the base of this PR to the `trunk` branch. We will want to use the update for the entire scaffold :)

**Comment by joesnellpdx** [comment]
@jacobdubail 

We want to have the filter for the theme.json updates in here -right? Or is that still up for discussion?

**Comment by joesnellpdx** [comment]
ohh... @jacobdubail I see how you are doing it now.. and that's actually pretty slick!!

**Pull request merged**